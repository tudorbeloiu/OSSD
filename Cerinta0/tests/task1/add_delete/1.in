15
3
15
1
2
11
8190
12
12
3
12
3
12
3
11
3
11
1
1
12
14
3
12
1
3
21
400
22
400
23
400
3
21
3
23
1
5
31
800
32
500
33
420
34
401
35
396
3
35
1
2
41
200
42
205
3
33
