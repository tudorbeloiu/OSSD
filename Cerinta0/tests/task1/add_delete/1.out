11: (0, 1023)
12: (0, 0)
11: (0, 1023)
11: (0, 1023)
12: (0, 1)
21: (0, 49)
22: (50, 99)
23: (100, 149)
22: (50, 99)
23: (100, 149)
22: (50, 99)
31: (100, 199)
32: (200, 262)
33: (263, 315)
34: (316, 366)
35: (0, 49)
22: (50, 99)
31: (100, 199)
32: (200, 262)
33: (263, 315)
34: (316, 366)
41: (0, 24)
42: (367, 392)
41: (0, 24)
22: (50, 99)
31: (100, 199)
32: (200, 262)
34: (316, 366)
42: (367, 392)
